### PR TITLE
Add default language and test

### PR DESCRIPTION
**Pull request by nguhoa**
Fix issue: https://github.com/microsoft/Power-Fx/issues/1195

### CODE REVIEW

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Comment by LucGenetier** [comment]
✅ No public API change.

**Review by MikeStall — APPROVED** [review]

File: src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryLanguage.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public static async ValueTask<FormulaValue> Language(EvalVisitor runner, EvalVis
 
         public static StringValue Language(EvalVisitor runner, IRContext irContext)
         {
-            return new StringValue(irContext, runner.CultureInfo.Name);
+            return new StringValue(irContext, string.IsNullOrEmpty(runner.CultureInfo?.Name) ? "en-US" : runner.CultureInfo.Name);
```
[the remark this answers is not in the document]
Comment (nguhoa) [reply src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryLanguage.cs:23]: as offline, this en-US is default for our error message as well.

**Pull request merged**